src/Web/ClientApp/src/app/components/application/list/list.component.html

```diff
@@ -41,7 +41,7 @@ <h1 class="title">Applications</h1>
 								<span *ngIf="channel.activeRevisionNumber"><strong>v{{channel.activeRevisionNumber}}</strong></span>
 								<span class="icon">
 									<!-- TODO: adapt color to channel status and add tooltip -->
-									<fa-icon [icon]="faCircle" [ngStyle]="{'color': 'green'}"></fa-icon>
+									<fa-icon [icon]="faCircle" [ngStyle]="{'color': channel.status === jobStatus.Running ? 'green' : 'red'}"></fa-icon>
```

Comment:
Maybe remove the comment and add a tooltip so the user would know what each color means?